generate-version.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
there are systems where `/bin/bash` does not exist